.eslintrc.js

```diff
@@ -0,0 +1,46 @@
+const eslintConfig = {
+  extends: [
+    'eslint:recommended',
+    'plugin:ava/recommended',
+    'plugin:import/errors',
+    'plugin:import/warnings',
+    'plugin:promise/recommended',
+    'standard',
+  ],
+  parserOptions: {
+    sourceType: 'script',
+  },
+  plugins: [
+    'ava',
+  ],
+  rules: {
+    'ava/no-import-test-files': 0,
+    'ava/no-ignored-test-files': 0,
+    indent: [
+      'error',
+      2,
+      {
+        CallExpression: {
+          arguments: 'first',
+        },
+        SwitchCase: 1,
+      },
+    ],
+    'no-console': 0,
+    strict: 'error',
+  },
+};
+
+eslintConfig.parser = '@typescript-eslint/parser';
+eslintConfig.parserOptions.sourceType = 'module';
+eslintConfig.extends.push(
+  'plugin:@typescript-eslint/eslint-recommended',
+  'plugin:@typescript-eslint/recommended',
+  'plugin:import/typescript',
+);
+
+eslintConfig.rules['comma-dangle'] = ['error', 'only-multiline'];
+eslintConfig.rules.semi = ['error', 'always'];
+eslintConfig.rules['space-before-function-paren'] = ['error', 'never'];
+
+module.exports = eslintConfig;
```

Comment:
suggestion: this can definitely be merged with the `eslintConfig` above. In fact, this can probably be a JSON file instead of a JS file (although I personally prefer it in `package.json` because I hate all of the dotfiles in the root directory).

Reply:
A lot of our repos have dotfiles instead of keeping multiple configs in `package.json`, so the idea here was to keep Packager consistent with other projects. A nice thing about using .js files for configuration is they allow [adding comments](https://github.com/electron/fiddle/blob/main/.eslintrc.js), so IMO that justifies having a bunch of dotfiles.